.github/workflows/backend-api-spec-update.yml

```diff
@@ -19,21 +19,27 @@ jobs:
       - uses: actions/checkout@v3
         with:
           ref: 'infra/update-backend-api-spec'
+          fetch-depth: 0
```

Comment:
```suggestion
          fetch-depth: 0 # we need a full history to make rebase with master
```

Reply:
good idea in aim to do fast commit via github 😀